usr.sbin/ppp/ppp.8

```diff
@@ -2166,13 +2166,15 @@ set device /dev/cuau0
 set speed 115200
 .Ed
 .Pp
-Cuad0 is the first serial port on
+.Pa cuau0
```

Comment:
Nice catch! The reader is definitely not going to find `/dev/Cuad0`. However, I don't think they're going to find `cuau0` either.
```suggestion
.Pa cuaU0
```
This needs changed on line 545,604,1209,1212,2165,2166,2529,2563,2564,2565, and 4063 too.

This one is a major and serious typo that would make these instructions unusable for someone who didn't already know, so I'm wondering if we can mfc this to the beta.